databricks_cli/pipelines/cli.py

```diff
@@ -208,19 +215,47 @@ def reset_cli(api_client, pipeline_id):
 @provide_api_client
 def run_cli(api_client, pipeline_id):
     """
-    Starts the execution of a delta pipelines run by starting the cluster and processing data.
+    [Deprecated] Use the "start" command instead.
+
+    Starts a pipeline update by starting the cluster and processing data.
```

Comment:
similar comment. It does not always start a cluster. I think we should not speak about whether/when clusters are started, terminated, or reused, because is a detail that will change in the future as the product evolves.

Reply:
Updated to "Starts a pipeline update."
Created https://databricks.atlassian.net/browse/SC-93969 for cli message updates.